backend/server.js

```diff
@@ -0,0 +1,126 @@
+const express = require('express')
+
+
+const postgresql= require(  "./postgresql");
+
+
+const app = express();
+
+let bodyParser = require('body-parser');
+const port = 3000
+
+app.use(express.json());
+app.use(express.urlencoded({ extended: true }));
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({extended: true}));
+
+
+
+
+app.post("/check/", async (req, res) => {
```

Comment:
Please make this a GET with path params instead of a POST request, as described in #8 . That's how it's assumed to be in api.py, so it saves time not having to redo the calls in that Python file as well.
See Route Parameters section in [https://expressjs.com/en/guide/routing.html](https://expressjs.com/en/guide/routing.html) for how to do that.